src/js/index.js

```diff
@@ -19,6 +19,30 @@ polyfill();
 $(document).ready(function(){
     $('.trust-cases').slick({
     });
+
+    $('.engine-cards').slick({
+        dots: false,
+        infinite: true,
+        speed: 300,
+        slidesToShow: 3,
+        slidesToScroll: 1,
+        autoplay: true,
```

Comment:
Are you sure we want to _autoplay_ the slider?
Because with it, the slider keeps moving behind the code panel after one card is opened.
I'd purpose:
1. remove the autoplay,
2. merge
3. if it's really needed, to do in a new PR: stop the autoplay when a card is opened, and resume it when closed.